core/src/apps/common/storage/device.py

```diff
@@ -234,3 +235,20 @@ def get_slip39_iteration_exponent() -> Optional[int]:
     The device's actual SLIP-39 iteration exponent used in passphrase derivation.
     """
     return common.get_uint8(_NAMESPACE, _SLIP39_ITERATION_EXPONENT)
+
+
+def get_sd_salt_auth_key() -> Optional[bytes]:
+    """
+    The key used to check authenticity of the SD card salt.
+    """
+    return common.get(_NAMESPACE, _SD_SALT_AUTH_KEY, public=True)
+
+
+def set_sd_salt_auth_key(auth_key: Optional[bytes]) -> None:
+    """
+    The key used to check the authenticity of the SD card salt.
+    """
+    if auth_key is not None:
+        return common.set(_NAMESPACE, _SD_SALT_AUTH_KEY, auth_key, public=True)
```

Comment:
Shouldn't we check here whether `len(auth_key) == SD_SALT_AUTH_KEY_LEN_BYTES` and raise `ValueError` instead?

Reply:
Done in both getter and setter. https://github.com/trezor/trezor-firmware/pull/526/commits/5a7e2fdb7a4fa87372bbd246eddfb4769b9da8b8